profile/src/v100/networks/network/profile_network.rs

```diff
@@ -136,17 +153,18 @@ mod tests {
                 Account::placeholder_mainnet(),
                 Account::placeholder_stokenet(),
             ]),
+            Personas::default(),
         );
     }
 
     #[test]
-    fn json_roundtrip_placeholder_stokenet() {
-        let sut = ProfileNetwork::placeholder_stokenet();
+    fn json_roundtrip_placeholder_mainnet() {
```

Comment:
ah annoying git diff, I swapped `json_roundtrip_placeholder_mainnet()` and `json_roundtrip_placeholder_stokenet() `